rust/macros/module.rs

```diff
@@ -145,7 +145,7 @@ fn param_ops_path(param_type: &str) -> &'static str {
 
 fn try_simple_param_val(
     param_type: &str,
-) -> Box<dyn Fn(&mut token_stream::IntoIter) -> Option<String>> {
+) -> FnTrySimpleParam {
```

Comment:
I disagree with clippy here. In any case if a type alias is used, I think it should be put in the same module.

Reply:
I tend to agree with Björn, especially if the alias is only used once or a few times.